test/extended/operators/olm.go

```diff
@@ -87,4 +88,22 @@ var _ = g.Describe("[Feature:Platform] OLM should", func() {
 			o.Expect(msg).To(o.Equal("IfNotPresent"))
 		}
 	})
+
+	// OCP-21082 - Implement packages API server and list packagemanifest info with namespace not NULL
+	// author: bandrade@redhat.com
+	g.It("Implement packages API server and list packagemanifest info with namespace not NULL", func() {
+		msg, err := oc.AsAdmin().WithoutNamespace().Run("get").Args("packagemanifest", "--all-namespaces", "--no-headers").Output()
+		o.Expect(err).NotTo(o.HaveOccurred())
+		var packageserverLine string
+		for _, line := range strings.Split(msg, "\n") {
+			if strings.Contains(line, "etcd") {
```

Comment:
What happens if `etcd` is no longer in the default package list? can we make this more generic so that we just look for some non-empty string rather than look for a specific operator that we don't necessarily control?

Reply:
I agree with you, I'm working in a more generic solution to avoid hard-code. I'll check the index of the first word in the first result line, if it's different from 0 it means that the previous chars are empty which could be considered an empty namespace in that case. What do you think about this implementation?

```
	g.It("Implement packages API server and list packagemanifest info with namespace not NULL", func() {
		msg, err := oc.AsAdmin().WithoutNamespace().Run("get").Args("packagemanifest", "--all-namespaces", "--no-headers").Output()
		o.Expect(err).NotTo(o.HaveOccurred())
		packageserverLines := strings.Split(msg, "\n")
		if len(packageserverLines) > 0 {
			packageserverLine := strings.Fields(packageserverLines[0])
			if strings.Index(packageserverLines[0], packageserverLine[0]) != 0 {
				e2e.Failf("It should display a namespace for CSV: %s [ref:bz1670311]", packageserverLines[0])
			}
		}
	})
```